crowdfunding_compassion/models/crowdfunding_project.py

```diff
@@ -87,6 +111,82 @@ class CrowdfundingProject(models.Model):
     owner_firstname = fields.Char(string="Your firstname")
     active = fields.Boolean(default=True)
 
+    @api.one
```

Comment:
api.one is deprecated, it would be good to remove it and make the method like the other computes with a loop :
```
for project in self:
    project.image_medium = project.cover_photo
....
```